lib/membrane_ffmpeg_swresample_plugin/converter.ex

```diff
@@ -135,17 +150,17 @@ defmodule Membrane.FFmpeg.SWResample.Converter do
 
     case flush!(state.native) do
       <<>> ->
-        {{:ok, end_of_stream: :output}, %{state | queue: <<>>}}
+        {[end_of_stream: :output], %{state | queue: <<>>}}
 
       converted ->
-        {{:ok, buffer: {:output, %Buffer{payload: converted}}, end_of_stream: :output},
+        {[buffer: {:output, %Buffer{payload: converted}}, end_of_stream: :output],
          %{state | queue: <<>>}}
     end
   end
 
   @impl true
-  def handle_prepared_to_stopped(_ctx, state) do
-    {:ok, %{state | native: nil}}
+  def handle_terminate_request(_ctx, state) do
+    {[terminate: :normal], %{state | native: nil}}
```

Comment:
People say that cleaning native could be entirely removed so `handle_terminate_request` probablye too?